src/main/java/teammates/ui/controller/InstructorFeedbackCopyAction.java

```diff
@@ -68,18 +70,21 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         // isError == true if an exception occurred above
         
         boolean omitArchived = true;
-        data.instructors = loadCourseInstructorMap(omitArchived);
+        HashMap<String, InstructorAttributes> instructors = loadCourseInstructorMap(omitArchived);
         List<InstructorAttributes> instructorList =
-                new ArrayList<InstructorAttributes>(data.instructors.values());
-        data.courses = loadCoursesList(instructorList);
-        data.existingFeedbackSessions = loadFeedbackSessionsList(instructorList);
+                new ArrayList<InstructorAttributes>(instructors.values());
+        List<CourseAttributes> courses = loadCoursesList(instructorList);
         
-        if (data.existingFeedbackSessions.isEmpty()) {
-            statusToUser.add(Const.StatusMessages.FEEDBACK_SESSION_ADD_DB_INCONSISTENCY);
-        }
+        List<FeedbackSessionAttributes> feedbackSessions = loadFeedbackSessionsList(instructorList);
+        FeedbackSessionAttributes.sortFeedbackSessionsByCreationTimeDescending(feedbackSessions);
+        
+        data.initWithoutHighlightedRow(courses, null, feedbackSessions, instructors, fs, null);
         
-        FeedbackSessionAttributes.sortFeedbackSessionsByCreationTimeDescending(data.existingFeedbackSessions);
         
+        if (data.getFsList().getExistingFeedbackSessions().isEmpty()) {
+            statusToUser.add(Const.StatusMessages.FEEDBACK_SESSION_ADD_DB_INCONSISTENCY);
+        }
```

Comment:
May be good to keep the `init` call as the last or near last operation in each `Action`.
In this condition, I recommend moving it above the `init` call and using the `feedbackSessions` available here instead of doing a `getFsList().getExistingFeedbackSessions()`; this will ensure that changes to the number of feedback sessions for whatever reason in the `init` call will not affect the result of this condition.